campaign_themes/antarctic.json

```diff
@@ -152,7 +153,21 @@
 						{ "color": "#52c9f4" }
 					]
 				}
-			}
+			},
+			"default": "white"
+		},
+		{
+			"id": "spreadsheet",
+			"children": [
```

Comment:
Not sure about the naming, but this is to create a set of fields. The CSS Vars are a flat list but the JSON file doesn't need to be, so this is to avoid adding a `spreadsheet-` prefix to all of these values.

Reply:
After discussing via chat with @Inwerpsel I'm removing this key and the flattenizing function, I propose we stop any refactors until we settle what the structure of this file will be via this ADR: https://docs.google.com/document/d/1eciWb_8FHF4qmwPGNtHFtmf9zGsDCF3bGnwJhcUx13M/edit# ,  when we have more input from the product side. cc @comzeradd